project-base/app/config/config_test.yml

```diff
@@ -1,53 +1,2 @@
-# This configuration is cached in the container (delete cache after changes).
```

Comment:
I would like to keep those two lines, these information are quite important :slightly_smiling_face: